packages/create-sitecore-jss/src/templates/nextjs/src/Layout.tsx

```diff
@@ -12,14 +17,21 @@ interface LayoutProps {
   layoutData: LayoutServiceData;
 }
 
+interface RouteFields {
+  [key: string]: unknown;
+  pageTitle: Field;
+}
+
 const Layout = ({ layoutData }: LayoutProps): JSX.Element => {
   const { route } = layoutData.sitecore;
 
+  const fields = route?.fields as RouteFields;
+
   return (
     <>
       <Scripts />
       <Head>
-        <title>{route?.fields?.pageTitle?.value.toString() || 'Page'}</title>
+        <title>{fields.pageTitle.value.toString() || 'Page'}</title>
```

Comment:
It's required, because you can't just access `route.fields.pageTitle.value`, you can have array under the field, or another item, not a field, so you need to provide information about the exact type